subprojects/docs/src/samples/userguide/initScripts/customLogger/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+tasks.create("compile") {
```

Comment:
:question: didn't we agree to use `task("compile") { ... }` here to let the Kotlin script do exactly and look the same as the Groovy script?